### ISSUE TITLE
[BUG] Loss of Focus When Macro is Used

### ISSUE THREAD

**Issue by kookma** [Jermolene/TiddlyWiki5#7001]
### Describe the bug (this bug related to Parameterized Transclusion)

Error is related to https://github.com/Jermolene/TiddlyWiki5/pull/6666

* When you use a macro with $edit-text, you experience loss of focus in 5.3.0p
* Testing in 5.2.4p and 5.2.3 does not show such issue

### Expected behavior

The edit-text shall be remained under focus when one uses it.

### To Reproduce

1. open https://tiddlywiki.com/prerelease/
2. create a new tiddler with below content

```
\define rawCode() <$edit-text class="twg-wide-edit"	tag=textarea field=text/>

<$tiddler tiddler="TestTiddler">

<div class="twg-grid-container">
   <<rawCode>>
</div>

<div class="twg-grid-column twg-grid-rendered">
   <$transclude/>
</div>
  

</$tiddler>
```

3. save and start entering something in edit box, it works as expected

**NOW**

4. open https://tiddlywiki.com/prerelease/parameterised-transclusions/
5. create a tiddler with above content
6. save and start entering something in edit box, by each character you enter, you will have a loss of focus and you have to use mouse and click in edit-text to get it again under focus.

### Desktop (please complete the following information):

- OS: Windows 10
- Browser: Edge 105/Chrome 105/FireFox

**Comment by kookma** [comment]
NOTE: If you do not use the **macro** and directly include the content of macro in the place you use it, there is no loss of focus.
Using **procedure**, you will the same issue!

**Comment by Jermolene** [comment]
Thanks @kookma that's very helpful. I'm investigating...

**Comment by Jermolene** [comment]
Hi @kookma in my testing both v5.2.3 and the current prerelease also fail for the first keystroke if the tiddler "TestTiddler" does not exist. The parameterised-transclusions branch is failing on subsequent keypresses (ie even if TestTiddler does exist).

**Comment by Jermolene** [comment]
Thanks again @kookma fix is upcoming.

**Comment by kookma** [comment]
> Hi @kookma in my testing both v5.2.3 and the current prerelease also fail for the first keystroke if the tiddler "TestTiddler" does not exist.

Confirmed.  If tiddler is not exist there is loss of focus in both version!

**Comment by Jermolene** [comment]
Fixed in b426724f35b247a3c0dfb1051751af7cc35ff9e3

**Comment by kookma** [comment]
Much appreciated!
